grammars/asciidoc.cson

```diff
@@ -84,6 +206,32 @@
     ]
   }
 
+  # Matches explicit paragraph styles
+  #
+  # Examples
+  #
+  #   [normal]
+  #   This is a normal paragraph.
+  {
+    'match': '^\\[((normal|literal|listing|TIP|NOTE|IMPORTANT|WARNING|CAUTION|partintro|comment|example|sidebar|source|music|latex|graphviz))\\]$'
+    'name': 'markup.explicit.asciidoc'
+    'captures':
+      '1': 'name': 'support.constant.asciidoc'
+  }
+
+  # Matches section templates
+  #
+  # Examples
+  #
+  #   [appendix]
+  #   This is an appendix.
+  {
+    'match': '^\\[((sect[1-4]|abstract|preface|colophon|dedication|glossary|bibliography|synopsis|appendix|index))\\]$'
```

Comment:
Again if you agree with the comments above we could combine block name patterns into a single one.